websites/G/Google Maps/presence.ts

```diff
@@ -0,0 +1,72 @@
+const presence = new Presence({
+		clientId: "993496501886136371",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+function lettersOnly(str: string) {
+	return str.replace(/[^a-zA-Z]/g, " ");
+}
+
+presence.on("UpdateData", async () => {
+	const privacy = await presence.getSetting("privacy"),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		};
+	if (document.location.pathname.includes("/place")) {
+		if (privacy) presenceData.details = "Viewing a place";
+		else {
+			const place = document.location.href,
+				indexes = [];
+			for (const [index, element] of Array.from(place).entries())
+				if (element === "/") indexes.push(index);
+
+			presenceData.details = `Viewing ${lettersOnly(
+				place.substring(indexes[4] + 1, indexes[5])
+			)}`;
+			presenceData.buttons = [
+				{ label: "View Place", url: document.location.href },
+			];
+		}
+	} else if (document.location.pathname.includes("/dir")) {
+		if (privacy)
+			presenceData.details = "Viewing directions from one place to another";
```

Comment:
Sounds rather conversation
```suggestion
			presenceData.details = "Viewing directions to a location";
```